pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/mergerollup/segmentgroupmananger/MergeRollupTaskSegmentGroupManagerProvider.java

```diff
@@ -27,20 +27,25 @@
  * Provider class for {@link MergeRollupTaskSegmentGroupManager}
  */
 public abstract class MergeRollupTaskSegmentGroupManagerProvider {
+
+  private static String _defaultMergeRollupTaskSegmentGroupManagerClassName =
+      "org.apache.pinot.plugin.minion.tasks.mergerollup.segmentgroupmananger.DefaultMergeRollupTaskSegmentGroupManager";
```

Comment:
Better way would be class.getName(), so that any refactoring efforst won't miss this.